kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CustomResourceTest.java

```diff
@@ -91,12 +91,13 @@ public void testCreate() throws IOException {
   @Test
   public void testCreateOrReplace() throws IOException {
     String jsonObject = "{\"apiVersion\": \"test.fabric8.io/v1alpha1\",\"kind\": \"Hello\"," +
-      "\"metadata\": {\"resourceVersion\":\"1\", \"name\": \"example-hello\"},\"spec\": {\"size\": 3}}";
+      "\"metadata\": {\"resourceVersion\":\"1\", \"name\": \"example-hello\"},\"spec\": {\"size\": 3, \"scope\": \"Namespaced\"}}";
```

Comment:
Hi, Why have you added `scope: Namespaced` in CustomResource's body? Is this intentional?

Reply:
If was intentional, although not needed, because the crd is namespaced in this case and I saw it while debugging the test.
If that bothers you it can be removed...

Reply:
Please correct me if I'm wrong, I think this is CustomResource's body not CustomResourceDefinitions's. The test assumes that CRD already exists. We're providing whether Custom Resource is `Namespaced` or not in `CustomResourceDefinitionContext` here:
https://github.com/fabric8io/kubernetes-client/blob/f1d695d59ce50a9067639df0b79855f03096adc8/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CustomResourceTest.java#L67

Reply:
Yes, indeed... my fault! I will correct that tonight.

Reply:
@rohanKanojia done, thanks for the sharp eye!

Reply:
Np :+1: Thanks for taking time to contribute to Fabric8 :heart: